src/config.rs

```diff
@@ -50,6 +50,8 @@ pub struct OnRamp {
     #[serde(rename = "linked", default = "Default::default")]
     // TODO validate that this is turned on only for supported onramps (rest, ws)
     pub(crate) is_linked: bool,
+    #[serde(rename = "error_required", default = "Default::default")]
```

Comment:
can we stick to `err` instead of `error`? It might be confusing for people which one to use.

Reply:
good catch!